services/cron/tasks_basic.go

```diff
@@ -171,6 +171,16 @@ func registerActionsCleanup() {
 	})
 }
 
+func registerSyncRepoLicenses() {
+	RegisterTaskFatal("sync_repo_licenses", &BaseConfig{
+		Enabled:    false,
+		RunAtStart: false,
+		Schedule:   "@annually",
```

Comment:
Why `annually`? :thinking:

Reply:
What does this sync do? Detect licence in repo? I would say run it at least weekly.

Reply:
As I understand it, it's simply a consistency check as the licenses will already be synced on any commit

Reply:
Maybe monthly then?

Reply:
I would make it a manual task ... as it re-scan all repos

and you do want to only do this if things got wrong ... for old stale repos where no new commit push to default branch are done

Reply:
A manual task is also necessary with both the cron task.